base/applications/rapps/appview.cpp

```diff
@@ -1299,7 +1299,7 @@ CAppsListView::AddApplication(CAppInfo *AppInfo, BOOL InitialCheckState)
             hIcon = ExtractIconW(hInst, szIconPath.GetString(), 0);
         }
 
-        if (!hIcon)
+        if (!hIcon || (hIcon == (HICON)(INT_PTR)1))
```

Comment:
```suggestion
        /* Use the default icon if none were found in the file, or if it is not supported (returned 1) */
        if (!hIcon || (hIcon == (HICON)(UINT_PTR)1))
```
(by "not supported" i mean, executable, DLL or icon file)